### PR TITLE
Maintain a list of surrogate splits

### PR DESCRIPTION
**Pull request by xulaus**

### CODE REVIEW

File: CHAID/CHAID.py [hunk 1]
```diff
@@ -199,7 +202,7 @@ class CHAID(object):
     split_titles : array-like 
         array of names for the independent variables in the data
     """
-    def __init__(self, ndarr, arr, alpha_merge=0.05, max_depth=2, min_sample=30, split_titles=None):
+    def __init__(self, ndarr, arr, alpha_merge=0.05, max_depth=2, min_sample=30, split_titles=None, split_threshold=0):
```
Comment (Rambatino) [inline CHAID/CHAID.py]: wouldn't it make sense to have a more realistic default?

Comment (xulaus) [inline CHAID/CHAID.py]: 0 disables the functionality, so is just saying its off by default

**Comment by Rambatino** [comment]
Is this a bug on their end?
https://www.quantifiedcode.com/app/project/gh:Rambatino:CHAID/diff/20a3cce7c4b95434cdb7e25ff2182a94f72248e8/2f6ee0d86ed0d8ffaa6fbf8bd362291cfc058d4d/issues?groups=code_patterns%3A2QdgB8ll%3Af0

**Comment by Rambatino** [comment]
Also, spec?

**Comment by xulaus** [comment]
thats not a bug on there end. `p` is considered a poor variable/method name because it is too short, can't be helped here though without breaking backward API compatibly.

**Comment by Rambatino** [comment]
👍

**Pull request merged**